packaging/convert2rhel.spec

```
@@ -124,6 +124,27 @@ install -m 0600 config/convert2rhel.ini %{buildroot}%{_sysconfdir}/convert2rhel.
 %attr(0644,root,root) %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Mon Feb 20 2023 Eric Gustavsson <egustavs@redhat.com> 1.2
+- Remove python 2.6 and RHEL 6 conversion capability
+- Check for kernel boot files after conversion
+- Allowing the overriding of kmod inhibitors
+- Remove shim-x64 special workaround
+- Always print the RHSM rollback task "title"
+- Check if org and key is passed
+- Improve logging and handling of RHSM facts
+- Change the label for FILE output to DEBUG
+- Add progress indicators for YUM and DNF modules
+- Refactor yum makecache to be separate from main checks
+- Man/help page to mention config file location
+- Add opt-in for convert2rhel RHSM data collection
+- Hide username and organization from logs and breadcrumbs
+- Fix conversion with "LANGUAGE" envar
+- Deduplicate log messages about imported keys
+- Update address for internet connection check
+- Improve repoquery parsing for loaded kernel check
```

This is just a description of what we did to fix a bug. It should rather be:
```suggestion
- Fix a traceback when checking if the latest kernel is loaded
```